cmd/minikube/cmd/delete.go

```diff
@@ -46,10 +48,27 @@ associated files.`,
 
 // runDelete handles the executes the flow of "minikube delete"
 func runDelete(cmd *cobra.Command, args []string) {
-	if len(args) > 0 {
-		exit.Usage("usage: minikube delete")
+	profileFlag, _ := cmd.Flags().GetString("profile")
+	deleteAllFlag, _ := cmd.Flags().GetBool("all")
```

Comment:
no need to get this here, right? if the flag was passed it will be on the `deleteAll`